fuzzers/xpath-31-function-catalog.xml

```diff
@@ -0,0 +1,18837 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
fontoxpath stores (big) test files in [test/assets/](https://github.com/FontoXML/fontoxpath/tree/master/test/assets). Please add instructions for downloading this file to the instructions for [running the fuzzer](https://github.com/FontoXML/fontoxpath/blob/master/CONTRIBUTING.md#running-the-fuzzer). This would make this file more maintainable. Maybe add a helpful error message when failing to load this document as well.

@DrRataplan do you agree with this?

Reply:
Why would you want to download that file? That would require writing code and thus raise maintenance cost. Also it is a point of failure and likely will lead to spurious build failures.

Reply:
Where did you find this file? We indeed try to push these very large files to outside of this repo to keep git clones fast. If this is mirrored anywhere, I prefer to add instructions to fetch it from somewhere.

Reply:
https://www.w3.org/TR/2017/REC-xpath-functions-31-20170321/function-catalog.xml linked from https://www.w3.org/TR/xpath-functions/.

The file is ~135kb minified, so it is unlikely to significantly impact cloning speeds IMHO.

Reply:
@MTimmerije could you add this to the asset download script (there are two now, in the `test` folder). That way we remove this file, it'll only get in the way of searching ad stuff. I do not expect us to make changes here. Don't forget to add it to the gitignore!